app/mailers/candidate_mailer.rb

```diff
@@ -15,8 +15,7 @@ def application_under_consideration(application_form)
       candidate: application_form.candidate,
       candidate_name: application_form.first_name,
       choice_count: application_form.application_choices.count,
-      rbd_date: application_form.application_choices.first.reject_by_default_at,
-      rbd_days: application_form.application_choices.first.reject_by_default_days,
+      rbd_days: TimeLimitConfig.limits_for(:reject_by_default).first.limit,
```

Comment:
The reason we use the database `reject_by_default_days` here is that it might change at some point, so we need to display the number of days that is true for that specific application.

Reply:
Okay great have reverted it 👍